boat-scaffold/src/main/java/com/backbase/oss/codegen/swift5/BoatSwift5CodeGen.java

```diff
@@ -0,0 +1,1070 @@
+package com.backbase.oss.codegen.swift5;
+
+import io.swagger.v3.oas.models.media.ArraySchema;
+import io.swagger.v3.oas.models.media.Schema;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.text.WordUtils;
+import org.openapitools.codegen.*;
+import org.openapitools.codegen.languages.Swift5ClientCodegen;
+import org.openapitools.codegen.meta.GeneratorMetadata;
+import org.openapitools.codegen.model.ModelMap;
+import org.openapitools.codegen.model.ModelsMap;
+import org.openapitools.codegen.model.OperationsMap;
+import org.openapitools.codegen.utils.CamelizeOption;
+import org.openapitools.codegen.utils.ModelUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.openapitools.codegen.utils.StringUtils.camelize;
+
+public class BoatSwift5CodeGen extends Swift5ClientCodegen {
```

Comment:
@DevAgani I've just noticed that a lot of this code was copied from `Swift5ClientCodegen` - I'm not sure if it should be done this way.

What exactly should be customised?

Reply:
@walaniam, Thank you for you keen look.

These changes were as a result of adopting the generator we were using locally. However, i have proceeded to make a considerable amount of changes as you can see from the last commit. 

I have made the changes